probabilistic/core/pdf.py

```diff
@@ -134,7 +138,7 @@ def _calculate_mid_price(options_data: DataFrame) -> DataFrame:
     Returns:
         the options_data DataFrame, with an additional column for mid-price
     """
-    options_data["mid_price"] = (options_data.bid + options_data.ask) / 2
+    options_data["mid_price"] = options_data["last price"]
```

Comment:
Can we get rid of `mid_price` completely and just use `last_price` everywhere instead?

Reply:
Done!